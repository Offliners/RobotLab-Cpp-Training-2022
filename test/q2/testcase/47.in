473
QuaKE
rhythm
FOlD
waRrANt
RECONSTRUcTIOn
HASTY
EArTHQUakE
dEsert
hunter
oRDiNariLy
mY21Olo80
anotheR
BINd
OVEREAt
moMENtUm
ShOe
maJority
G6oB39
CuT
DeBt
cHroNic
HEADInG
pITch
GradUAte
mODErAte
ComEt
MeNTOr
AfteR
RAG
EmBASSY
EpIgRAm
SAL8A5E
CORRUPT
COstUmE
PAvEMENT
eAR
CHeErLEADER
DEVASTATING
eStabliSh
MARRIAGE
regulatE
sexual
pOssibilIty
ashaMeD
pORTraIT
1AmP
PICK-UP
MASSACRE
caRROt
DIsMAy
purely
laY
outset
USEFuL
FARE
eNRoll
mAnAgeR
HUsBAnD
oLd
REAsoN
GoverN
INviting
CRUMBLE
ANYMORE
doMAin
EXPERt
DIFFEReNT
docUmEntArY
TEmPer
cable
efficiEncy
SUDDEN
RHETOrIC
DIRECTORY
PARaDISe
ImmiGraNt
preFeR
cONfIDenT
MILD
IF
FAVOrItE
melLow
DISeAsE
tWIst
HOT
InpuT
PeNICILLIN
tYrant
cAse
8o9993r
diSAPPEar
rAnSom
pictUresque
StrucTure
useD
GlUe
bEAst
sHOp
OrGanIZe
ApPROVal
emPirical
tRANSFER
oRIGINALITy
DECeIvE
BOdYgUaRd
armchaIr
mushroom
rEPEtITION
drug
cIrCULaTE
childhood
EYESIGHT
REFLECT
PHeNOMENON
ImmINENT
17m1g19tE
SeA
ROYALtY
PeNnY
TILL
COMMENT
bEckOn
DILEMMA
tHEsIS
PROFITAbLE
c2ew
evolution
PRODuCTIve
cracKdOWN
DOwNToWN
giN
O
2187C906
liberal
pA5sivE
IRRESPONSIBLE
cOOkIe/cooKY
EdITION
ouTRaGeoUs
buffEt
producTivity
justIcE
INfErEnCE
POSTMAN
CONtRIBUTE
ideology
TABLE
gas
EaSY
airwaY
EaRNINgS
PLaN
INFEREN2E
MAINTAIN
SUFFERINg
delight
messeNgER
taBLEt
2el03000062
POWER
wAke
AvOID
REPRESeNT
NEGOTIABLe
forgE
intEnSitY
CompAny
station
wheNevEr
hesItANTly
COuntRY
deliberation
deviCE
meLODY
HOStILITY
sOUth
PartnEr
october
5L3V79
CONTAINER
incentive
contracT
AVERaGE
posseSs
ReForm
TangerINE
infER
STACK
PURPLE
cHeMical
S5780v4
MAGICAL
SHELF
reAdiNESs
TURTLE
lEak
nigHtmare
tARiFf
FLEE
ANxIOuS
reject
cONstitutE
deAFEn
physics
HEAVyWEIGHT
oversEas
poet
jAnuAry
MARS
BesIegE
aMaZiNg
b876E3
SIX
POLE
seNd
scareD
bleEd
deViation
compare
vERy
halt
annivErsAry
PaNIc
DOSE
PORTer
COiN
INDIGnAtiOn
Irritate
tiRESOME
m36e
scAndAl
BOLD
NICKNAME
moon
CRuDE
aptitUde
CONFORM
Plea1e1
wELCOme
cucumber
COsmetIC
sTEreoTYPe
AWFUL
AsSOCiatIon
teach
inABI0Ity
aPPRENTicE
DRAG
037t92
HurRy
ChECKBOOK
Job
UpSET
InFANT
iNFantry
DESPITE
Fo2gEtful
9229O9
VIneYArd
MEChANICAL
CoMMITMENT
DISTAnt
AwAKEN
tRAIN
bEing
Shed
HaUL
lord
TIDe
July
BREEZE
reCogNition
iNdependEnt
RElIABLE
PIlgRIM
complication
240K
SUmmER
fax
check-out
CuRIOUs
DONAtIO5
LAZY
PrOfEsS
gOsSip
fAShiON
BooTh
ELEVeN
honest
classIfiCatiON
UnAnIMOUSLY
TIP
goose
RespOnD
game
EConomy
DefeNSIvE
joke
NEWLYWEd
heROIne
bOtANY
COlUMnist
iNstAllAtIoN
PRESENCE
fRee
tOUrIST
rOBIN
MaSTErPIece
accIdEnt
hEiresS
SuRVIVe
VARIATION
baThTub
BIoLogY
TRIP
TOAST
lounge
LOaf
FISHErMAn
bRew
drUgSTORe
prominent
mistrEss
sPARKLe
IrONY
photOgrapher
ReacT
TAX
bOrn
C
eVer
pSyChologicAl
pHoNe
clear
LANDSLIDE
ACCOUNTANT
ExECUTIVe
ideNtIfiCation
CHAPTER
existence
BiCyCLE/BIKE
sMallPox
allegE
WHeREABoUTS
suitcaSe
RacIal
extension
EGO
ENLIgHTEN
CEntURY
HOMe
cAsuAltY
GYMNASIUM/GYM
hearsay
fLour
VARIABLe
CAMPaign
pErseverance
laMA
sword
fUrNitUrE
PREMATURE
amuse
MAID
fEed
ground
tHUNDER
EconOmisT
PENTAGON
controLLEr
legally
consequent
dramatic
dWArF
THiGH
DISTrIBUTION
foreSt
GROcery
BAnK
advance
FOrCE
SUPERFLuOUS
praiseWorThY
go
MARVEL
rib
rEbEllIon
ADMISsIoN
plAnetARY
beer
RePrESENTAtIVE
APpEAL
CURRICULUM
tasK
TWENTY
INdOOR
T726u7
upbRInGing
sACK
lifeboat
line
awKwardlY
cHIldbiRth
D94849
taNK
asleep
REMEMBER
RUNNER
bLoOD
ERoDe
largely
COUPON
FAVORAbLE
coMpeTEnCE
cOFFiN
adApt
REAch
COMbINE
LiKe
APPLIANcE
SUBJECTIVE
blinD
versatIle
AUTHoR
StReNGTHEN
NUCLEUS
LIVE
cOMpUter
nUtrItion
04p30053I98
gloW
DrIvE
BiZArRE
dadd7/dad
8A0
HUNDRED
HAWK
MondaY
sEAsON
interpret
ORIGINaL
AuDITORIUM
hiSToRy
design
FOuNDer
rAncher
sanction
COmputeRIze
9Efer
terrible
planET
SURE
conTRAST
MeasLeS
compaNionship
COMMUtEr
cUStomEr
HOnORabLe
instead
TOFU
INTerESTINg
RaNge
NATURALLY
scientific
color
coUGh
weatheR
HApPINESS
narRate
cONTRIbUTIOn
CaREFuL
spAGheTTi
prepAre
