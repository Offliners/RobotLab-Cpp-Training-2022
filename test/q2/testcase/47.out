v. 震動
n. 節奏
n./v. 摺疊；交疊
n. 搜查令
n. 重建(物)
adj. 匆忙的
n. 地震
n. 沙漠
n. 獵人
adj. 正常地；普通地
Unknown
adj./pron. 另一個
v. 捆；綁
v. 暴食
n. 動量；動向
n. 鞋
n. 多數
Unknown
n./v. 切
n. 債
adj. 慢性的
n. 標題
n. 瀝青
v./n. 畢業；大學畢業生
adj. 中等的
n. 彗星
n. 導師
prep./adv./adj. 在…之後；以後的
n. 破布
n. 大使館
n. 雋語
Unknown
adj./v. 腐敗的；敗壞
n. 服裝；戲服
n. 人行道
n. 耳朵；聽力
n. 啦啦隊隊長
adj. 毀滅性的
v. (n)建立
n. 結婚
v. 限制；控管
5adj. 男女之問的
n. 可能性
adj. 感到羞恥的
n. 肖畫像
Unknown
n. 擴音器
n. 大屠殺
n. 胡蘿蔔
v. 使驚慌
adv. 完全地
v. 放；安排；下蛋
n. 開始
adj. 有用的
n. 車費；伙食
v. 使加入；註冊
n. 經理；管理人
n. 丈夫
adj. 老的；舊的
n. 理由
v. 統治；治理
adj. 吸引人的
v. (使)碎成細屑
adv. 再也(不)
n. 領域
adj./n. 熟練的；專家
adj. 不同的
n. 記錄片
n. 脾氣
n./v. 電纜；有線電視
n. 效率
adj. 突然的
n. 修辭學
n. 工商人名錄
n. 天堂
n. 移民
v. 寧願選擇
adj. 有自信的
adj. 溫和的；順口的
conj.  如果
adj./n. 特別喜愛的
adj. 香醇的；柔和的；老練的
n. 疾病
n./v. 彎曲；扭曲
adj. 熱的；熱情的
n./v. 輸入
n. 盤尼西林
n. 暴君
n. 事實；實例
Unknown
v. 消失
n. 贖金
adj. 似畫的
n. 結構
adj. 舊的
n./v. 膠水；黏牢
n. 野獸
n. 商店
v. 組織
n. 同意；認可
adj. 憑經驗的
n./v. 調動
n. 創新；創造力
v. 欺騙
n. 侍衛
n. 扶手椅
n./v. 香茹；蘑蘋狀的束西
n. 重覆
n./v. 藥品；毒品
v. 循環；流通
n. 童年時期
n. 視力
v. 反射
n. 現象
adj. 逼近的
Unknown
n. 海洋
n. 皇室成員
n. 一分錢
prep./conj. 直到
n./v. 評論
v. 召喚
n. 困境
n. 論文
adj. 有益的
Unknown
n. 演化(論)
adj. 多產的
n. 取締
adv./adj./n. 在城市的商業區；鬧區
n. 杜松子酒
Unknown
Unknown
adj. 心胸寬闊的
Unknown
adj. 不須負責任的
n. 甜餅乾
n. 發行數；版本
adj. 駭人的；暴虐的
n. 自助餐；快餐
n. 生產力
n. 公平；正義
n. 推論
n. 郵差
v. 捐獻；貢獻
n. 意識形態
n. 桌子
n./v. 氣體；瓦斯
adj./adv. 容易的
n. 航空公司；航線
n. 收入；工資
n./v. 計劃
Unknown
v. 保持
n. 苦難
n./v. 喻快
n. 使者；信差
n. 藥片
Unknown
n. 權力
v. 醒來
v. 避免；躲開
v. 描寫
adj. 可協商的
v. 打(鐵等)
n. 強烈；強度
n. 公司；同伴
n. 車站
adv./conj. 無論什麼時候；每當
adv. 遲疑地
n. 國家；鄉下
n. 深思
n. 裝置
n. 旋律
n. 敵意
adj./adv./n. 南方的
n. 伙伴
n. 十月
Unknown
n. 容器
n./adj. 刺激
n./v. 契約
adj./n./v. 平均的
v. 擁有
v. 改革
n. 橘子
v. 推斷
v. 使成堆
adj./n. 紫色的
adj./n. 化學的
Unknown
adj. 魔法的
n. 架子
n. 準備就緒；迅速
n. (海)龜
v. 漏
n. 惡夢
n. 關稅
v. 逃走；逃離
adj. 焦慮的
v. 拒絕；否認
v. 構成；制定
v. 使聽不見
n. 物理學
n./adj. 重量級拳手；重的
adj./ad v. 來自海外的
n. 詩人
n. 一月
n. 戰神；火星
v. 包圍
adj. 令人驚訝的
Unknown
adj./n. 六
n. 南極或北極
v. 送
adj. 害怕的
v. 流血
n. 偏離
v. 比較
adv. 非常
v./n. 停止；暫停
n. 週年紀念
n./v. 恐慌
n. (藥物等的)一劑
n. 搬行李工人
n. 硬幣
n. 憤慨
v. 激怒
adj. 令人厭煩的
Unknown
n. 醜聞
adj. 英勇的；大膽的
n. 綽號
n. 月亮
adj. 未加工製造的
n. 資質
v. 遵從
Unknown
n./v. 歡迎
n. 黃瓜
n./adj. 化妝品；美容的
n. 老套
adj. 糟透的
n. 協會；聯想；合作
v. 教導
Unknown
n. 學徒
v. 拖曳
Unknown
n./v. 急忙
n. 支票簿
n. 工作
adj./v. 心煩的；使生氣
n. 嬰兒
n. 步兵
prep. 儘管
Unknown
Unknown
n. 葡萄園
adj. 機械的
n. 承諾
adj. 遙遠的
v. 意識到；使覺醒
n. 火車
n. 存在；生物
v. 使流出
v. 拖
n. 統治者
n. 潮汐
n. 七月
n./v. 微風；和風
n. 承認
adj. 獨立的
adj. 可信賴的
n. 朝聖者
n. 糾葛
Unknown
n. 夏天
v./n. 傳真；傳真機
n. 檢查；付款退租
adj. 好奇的
Unknown
adj. 懶散的
v. 自稱
n./v. 流言輩語
n./v. 時尚；風氣
n. 售貨棚
n. 十一
adj. 誠實的
n. 分類
adv. 全體一致地
n./v. 小費
n. 鵝；傻瓜
v. 回答
n. 遊戲；競賽
n. 節約；經濟
adj. 防護的；保衛的
n./v. 玩笑
n. 新婚的人
n. 女英雄
n. 植物學
n. 專欄作家
n. 安置
n. 出席
adj. 自由的
n. 觀光客
n. 知更鳥
n. 傑作；名著
n. 事故
n. 女繼承人
v. 悻存
n. 變化
n. 浴缸
n. 生物學
n. 旅行
n./v. 吐司；烤；烘
n./v. 休息室；懶洋洋地坐或站
n. 一條麵包；遊蕩
n. 漁民；漁夫
v. 沖泡；醞釀
n. 雜貨店
adj. 突起的
n. 女主人
n./v. 閃耀
n. 反語
n. 攝影者
v. 反應
n./v. 稅；對…微稅
adj. 出生的
Unknown
adv. 永遠
adj. 心理(上)的
n./v. 電話
adj./v. 清楚的
n. 山崩
n. 會計師
adj. 執行的
n. 識別；身分證明
n. 章，回
n. 存在
n. 腳踏車
n. 天花
v. 聲稱
adv. 在哪兒
n. 手提箱
adj. 種族的
n. 伸長；延期
n. 自我
v. 敵發
n. 世紀
n./adv. 住家；家庭
n. 傷亡者
n. 體育館
n. 謠言
n. 麵粉
adj. 易變的
n./v. 競選活動；戰役
n. 不屈不撓
n. 喇嘛
n. 劍
n. 傢俱
adj. 過早的
v.(n. )給～提供娛樂
n. 少女
v./n. 餵食
n. 地面
n./v. 雷聲；打雷
n. 經濟學者
n. 五角形
n. 控制者
adv. 法律上
adj. 因…的結果而起的
adj. 戲劇性的
n./adj. 侏儒；矮小的
n. 大腿
n. 分配；配給物
n. 森林
n. 食品雜貨店
n. 銀行
n./v. 推進；將…提前
n./v. 強迫；力量
adj. 過多的
adj. 值得讚揚的
v. 去；從事
n. 令人驚奇或驚喜的事物
n. 肋骨
n. 造反
n. 加入許可；承認
adj. 行星的
n. 啤酒
adj./n. 象徵的；例子
n./v. 吸引；呼籲
n. 課程
n. 任務
adj./n. 二十
adj. 室內的
Unknown
n. 養育
n. 大袋子
n. 救生艇
n./v. 線條；畫線於
adv. 笨拙地
n. 分娩
Unknown
n. 大容器
adj. 睡著的
v. 記得
n. 奔跑的人
n. 血液
v. 侵蝕
adv. 大部分
n. 優待券
adj. 贊同的；適合的
n. 能力
n. 棺材
v. 使適應
n./v. 抵達
v. 結合
prep. 像；和·一樣
n. 器具；裝備
adj. 主觀的
adj./v. 瞎的；使看不見
adj. 多才多藝的
n. 作者
v. 加強
n. 中心；核心
adj. 活的
n. 電腦
n. 營養；營養學
Unknown
v./n. 發光；發熱
n./v. 車程；駕駛
adj. 古怪的
Unknown
Unknown
adj./n. 一百
n. 鷹
n. 星期一
n. 季節
v. 解釋；翻譯
adj./n. 最初的；原著
n. 禮堂
n. 歷史
n./v. 設計
n. 創立者；締造者
n. 農場主人
n. 批准
v. 使電腦化
Unknown
adj. 可怕的
n. 行星
adj. 確信的
n./v. 對比
n. 麻疹
n. 交誼
n. 通勤者
n. 顧客
adj. 榮譽的
adv. 反而
n. 豆腐
adj. 有趣的
n. 區域
adv. 自然地
adj. 科學的
n./v. 顏色；彩色
v. 咳嗽
n. 天氣
n. 快樂
v. 敘述
n. 貢獻；捐獻
adj. 仔細的
n. 義大利麵條
v. 準備
